microsoft-edge/webview2/release-notes/index.md

```diff
@@ -120,6 +123,299 @@ description
+
+* [ICoreWebView2SaveFileSecurityCheckStartingEventHandler](/microsoft-edge/webview2/reference/win32/icorewebview2savefilesecuritycheckstartingeventhandler?view=webview2-1.0.2849.39&preserve-view=true)<!-- Win32-only -->
+
+---  
+
+
+<!-- ------------------------------ -->
+#### Bug fixes
+
+
+<!-- ---------- -->
+###### Runtime-only
+
+* Fixed a **Download** dialog focus issue when pressing **Tab** or **Shift+Tab** to switch into the Webview2 control.
+
+
+<!-- ---------- -->
+###### SDK-only
+
+* Using `CoreWebView2.AddWebResourceRequestedFilter` without a `CoreWebView2WebResourceRequestSourceKinds` parameter is now deprecated.  See the .NET [CoreWebView2.AddWebResourceRequestedFilter Method](https://go.microsoft.com/fwlink/?linkid=2286319).<!-- todo: link to Win32 & WinRT?  applies to them? -->
+
+<!-- end of Oct 2024 Release SDK -->
+
+
+<!-- ====================================================================== -->
+## 1.0.2895-prerelease
+
+Release Date: October 21, 2024
+
+[NuGet package for WebView2 SDK 1.0.2895-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.2895-prerelease)
+
+For full API compatibility, this Prerelease version of the WebView2 SDK requires the WebView2 Runtime that ships with Microsoft Edge version 131.0.2895.0 or higher.
+
+
+<!-- ------------------------------ -->
+#### Experimental APIs
+
+The following Experimental APIs have been added in this Prerelease SDK.
+
+
+<!-- ---------- -->
+###### `RestartRequested` event when WebView2 needs to restart
+
+Added a new `RestartRequested` event.  The `RestartRequested` event is raised whenever WebView2 needs to restart to apply updates or configuration changes.  You can use this API to detect when WebView2 needs to restart, and take appropriate actions.  The `Priority` property of the `RestartRequested` event arguments indicates the priority of the restart request, which can either be `HIGH` or `NORMAL`:
+* `High` priority indicates that the app should prompt users to restart as soon as possible.
+* `Normal` priority indicates that the app should remind users to restart, on a best-effort basis.
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* `CoreWebView2Environment` Class: 
+    * [CoreWebView2Environment.RestartRequested Event](/dotnet/api/microsoft.web.webview2.core.corewebview2environment.restartrequested?view=webview2-dotnet-1.0.2895-prerelease&preserve-view=true)
+
+* `CoreWebView2RestartRequestedEventArgs` Class:
+    * [CoreWebView2RestartRequestedEventArgs.Priority Property](/dotnet/api/microsoft.web.webview2.core.corewebview2restartrequestedeventargs.priority?view=webview2-dotnet-1.0.2895-prerelease&preserve-view=true)
+
+* [CoreWebView2RestartRequestedPriority Class](/dotnet/api/microsoft.web.webview2.core.corewebview2restartrequestedpriority?view=webview2-dotnet-1.0.2895-prerelease&preserve-view=true)<!-- todo: enum, or class? -->
```

Comment:
```suggestion
* [CoreWebView2RestartRequestedPriority Enum](/dotnet/api/microsoft.web.webview2.core.corewebview2restartrequestedpriority?view=webview2-dotnet-1.0.2895-prerelease&preserve-view=true)
```
enum, or class?  see other tabs, or dest link, or prerelease equiv link
@victorhuangwq

Reply:
I think for .NET, these things are technically classes instead. @peiche-jessica

Reply:
worst case: publish relnotes, then test (easy at that time), then follow-on PR if needs per-platform fix

for now, keeping .md as-is, keeping Comment open

Reply:
It should be enum. I'd share more in the chat.

Reply:
result:
```
* [CoreWebView2RestartRequestedPriority Enum](/dotnet/api/microsoft.web.webview2.core.corewebview2restartrequestedpriority?view=webview2-dotnet-1.0.2895-prerelease&preserve-view=true)
    * `CoreWebView2RestartRequestedPriority.High`
    * `CoreWebView2RestartRequestedPriority.Normal`
```